app/cafe.py

```diff
@@ -0,0 +1,18 @@
+from datetime import date
+from app.errors import (NotVaccinatedError,
+                        NotWearingMaskError,
+                        OutdatedVaccineError)
```

Comment:
Make sure that all errors are placed on a new line so that there are no imports on the same line with opening and closing parenthesis to improve the readability

Reply:
.\app\cafe.py:3:80: E501 line too long (86 > 79 characters).
have to place on several lines to avoid requerments of flake8

Reply:
Place each import on the new line